### DISCUSSION TITLE
请问关于动画与时间的一点问题？

### DISCUSSION THREAD

**Discussion by rocketman123456** [BoomingTech/Piccolo#387] [Announcements]
在研究作业3的过程中，我发现游戏中的动画时间参数是一直不断循环的，并不会因为切出game mode或者是取消focus就暂停更新。请问之后是不是准备添加一个虚拟时钟，来进行动画时间的管理，可以更好的实现动画的启动，更新，停止，以及对时间进行缩放等功能？

**Comment by yijiang233** [comment]
目前editor模式下动画会暂停tick，但alt切出后仍会tick。游戏中由像动画这样的系统需要暂停tick的系统还有很多，一般会统一管理
